rosdep/python.yaml

```diff
@@ -6717,6 +6717,62 @@ python3-gnupg:
   openembedded: [python3-gnupg@meta-python]
   opensuse: [python3-python-gnupg]
   ubuntu: [python3-gnupg]
+python3-google-auth:
+  debian:
+    '*': null
+    bullseye:
+      packages: [python3-google-auth]
+  ubuntu: [python3-google-auth]
+python3-google-auth-pip:
```

Comment:
We should not have a pip and non-pip rule.

Reply:
Sorry for being late, I have a trouble 
> The reason why I want to add both python3-google-auth and python3-google-auth-pip is that python3-google-cloud-pubsub-pip (https://github.com/ros/rosdistro/pull/32897) requires google-auth>=2.x. If not requires its version, python3-google-cloud-pubsub doesn't work well.

and need pip version of google-auth.